src/main/java/net/mcreator/ui/modgui/ItemGUI.java

```diff
@@ -120,7 +120,7 @@ public class ItemGUI extends ModElementGUI<Item> {
 	private final JCheckBox isMeat = L10N.checkbox("elementgui.common.enable");
 	private final JCheckBox isAlwaysEdible = L10N.checkbox("elementgui.common.enable");
 	private final JComboBox<String> animation = new JComboBox<>(
-			new String[] { "none", "eat", "block", "bow", "crossbow", "drink", "spear" });
+			new String[] { "none", "eat", "block", "bow", "crossbow", "drink", "spear", "spyglass" });

```

Comment:
The NONE animation doesn't swing for items, so I dont see a method of implementing a swing animation other than having to make a whole new one. Will keep it as-is since this PR mainly focuses on ranged items, I only threw the spyglass animation there since it was just a matter of adding one new word to the code.